wiki/osu!_Program_Files/en.md

```diff
@@ -76,18 +69,16 @@ The format is `{Beatmap number} {Artist} - {Song Title}`.
 
 Please note that *ancient* beatmaps folders during the beginnings of osu! do not have numberings and does not follow the format. Some examples would be "[discoprince](http://osu.ppy.sh/s/1)" or "[avgn](http://osu.ppy.sh/s/66)". Special mention goes to "[tutorial](http://osu.ppy.sh/s/3756)" because every osu! player will have it when osu! is installed using the osu!installer. Also, manually created/graveyard .osz/.zip songs folders can be counted as well.
 
-Hidden Folders
---------------
+### Hidden Folders
 
-### Data
+#### Data
 
 **osu!data files.** Should not be tampered.
```

Comment:
I find it funny how `**osu!data files.**` is **bold** but `Should not be tampered.` isn't